src/cpu_regs.h

```diff
@@ -144,11 +148,13 @@
 #define EHID1_EN_LFSR_STALL_RS6                  BIT(62)
 #define EHID1_EN_LFSR                            BIT(63)
 
-#define SYS_IMP_APL_HID3                  sys_reg(3, 0, 15, 3, 0)
-#define HID3_DISABLE_ARBITER_FIX_BIF_CRD  BIT(44)
-#define HID3_DEV_PCIE_THROTTLE_LIMIT_MASK GENMASK(62, 57)
-#define HID3_DEV_PCIE_THROTTLE_LIMIT(x)   ((ULONG(x)) << 57)
-#define HID3_DEV_PCIE_THROTTLE_ENABLE     BIT(63)
+#define SYS_IMP_APL_HID3                                   sys_reg(3, 0, 15, 3, 0)
+#define HID3_DisDcZvaCmdOnly                               BIT(25)
```

Comment:
Please reformat this to uppercase with underscores like the other flags (same with all the others).